cli/src/main.rs

```diff
@@ -106,8 +110,6 @@ fn main() {
 
                 if args.check && check_if_diff(None, &original, &formatted, true) {
                     exit(1)
-                } else {
-                    print!("{formatted}")
```

Comment:
Accidentally removed?
This will break the `stdin` mode

Reply:
heh i have just spent the last 30 min figuring out why i broke stdin... :laughing: